deployment/v3/external/landing-page/install.sh

```
@@ -106,37 +106,47 @@ function landing_page() {
   INJIWEB=$(kubectl get cm global -o jsonpath={.data.mosip-injiweb-host})
   INJIVERIFY=$(kubectl get cm global -o jsonpath={.data.mosip-injiverify-host})
 
-select_index_template # calling function
+  select_index_template
+
+  # Build helm args array
+  helm_args=(
+    -n $NS install landing-page mosip/landing-page
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)